src/services/baseService.ts

```diff
@@ -124,7 +125,7 @@ export abstract class BaseService {
    * Get the access token from credentials token cache
    */
   protected getAccessToken(): string {
-    return (this.credentials.tokenCache as any)._entries[0].accessToken;
+    return (this.credentials.tokenCache as SimpleFileTokenCache).first().accessToken;
```

Comment:
It's not always going to be your `SimpleFileTokenCache`.  You should normalize to the token cache interface.